c10/core/TensorImpl.cpp

```diff
@@ -15,6 +15,14 @@ C10_DEFINE_int64(
     "The maximum memory in bytes to keep on shrink, if the difference between "
     "tensor sizes is bigger than this then tensor will be reset.");
 
+C10_DEFINE_string(
```

Comment:
Uhhhhhh... I would *not* make this a gflag lol. (That's what `C10_DEFINE_string` does; it doesn't create a string constant, if that's what you thought it does...)